### ISSUE TITLE
Placas BR detectando letra D como O

### ISSUE THREAD

**Issue by igtoth** [openalpr/openalpr#606]
A maioria dos reconhecimentos com placa D, estão sendo reconhecidos com O. Existe como arrumar isso?

**Comment by sandropf** [comment]
Mesmo problema. Além disso tenho problemas com Q e O.
Tente deixar o strict como 0... Melhora um pouco para mim.
Mas ainda sim, imagino que a melhor solução seja deixar a imagem mais próxima da placa, assim o algoritmo pode perceber melhor as curvas.

Ainda não fiz o teste, mas imagino que ajude.
Outra coisa que talvez ajude seja treinar para o ângulo específico com bastante imagens com D, O e Q. ainda não tentei isso, mas deve ajudar bastante.

**Comment by johnseeder** [comment]
Alguém conseguiu resolver a detectção de placas de moto?

**Comment by psybox** [comment]
Sobre as motos @johnseeder .

Usa 'br2' na configuração ele detecta porém o programa fecha sozinho depois de um tempo de uso.

**Comment by diegguinholima** [comment]
@psybox @sandropf  Vocês conseguiram fazer funcionar a detecção de placa de moto? No meu caso só consigo se a placa estiver bem próxima, e mesmo assim não pega a placa corretamente, na maioria dos casos detecta apenas os números.

**Comment by psybox** [comment]
sim, mas como mencionei ele da um erro e para o programa apos algum tempo.
